### PR TITLE
Frontend prod prep

### PR DESCRIPTION
**Pull request by chamthesleeptalker**
Added map interaction for every search

### CODE REVIEW

File: client/components/AppMap.vue [hunk 1]
```diff
@@ -50,30 +56,65 @@ export default {
...
+    //   // use the mapObject to call native Leaflet methods
+    //   mapObject.on("moveend", function(e) {
+    //     console.log(e);
+    //   });
+    // },
+    // onUpdateBounds({ bounds }) {
+    //   // assign the center coordinates to the data properties
+    //   this.bounds = bounds;
+    //   console.log(bounds);
+    // }
```
Comment (lkpanganiban) [inline client/components/AppMap.vue]: will these be used in the future? maybe we can remove these.

File: client/pages/index.vue [hunk 2]
```diff
@@ -192,10 +235,15 @@ export default {
...
-        // console.log("fetchSearchFunction - response JSON", data.features);
+
         this.filteredData = data;
         this.isFetching = false;
         this.error = null;
+
+
+        // console.log("data", data);
+        // console.log("data.features", data.features);
+
```
Comment (lkpanganiban) [inline client/pages/index.vue]: maybe we can also remove thes?

File: client/components/AppMap.vue (hunk 1 above)
Comment (chamthesleeptalker) [reply client/components/AppMap.vue]: I removed these lines for clarity.

File: client/pages/index.vue (hunk 2 above)
Comment (chamthesleeptalker) [reply client/pages/index.vue]: I removed these lines for clarity and other logs made for check.

**Pull request merged**